shared/chat/inbox/container.js

```diff
@@ -223,7 +224,9 @@ const mapStateToProps = (state: TypedState, {isActiveRoute, routeState}: OwnProp
     isLoading: inboxGlobalUntrustedState === 'loading' || state.chat.get('inboxSyncingState') === 'syncing',
     neverLoaded: inboxGlobalUntrustedState === 'unloaded',
     _user: Constants.getYou(state),
-    inSearch: state.chat.get('inSearch'),
+    showNewConversation: state.chat.get('inSearch') ||
+      // $FlowIssue this is a string
```

Comment:
Actually, this can be null -- If you navigate to the Chat tab on an initial load, it'll crash. This should be something like `... || (selectedConversationIDKey && Constants.isPendingConversationIDKey(selectedConversationIDKey))`